samples/sample_script.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env sh
+
+# set -xv
+
+# show usage and quit
+function usage()
+{
+  printf "usage: %s <command>\n" $(basename $0) >&2
+  printf "\n" >&2
+  printf "    <command> is one of: download, remove\n" >&2
+  printf "    <sample_name> is one of: iris, gsod, metriccentury\n" >&2
+  printf "\n" >&2
```

Comment:
Should this block be deleted?  It looks deprecated.

Reply:
I think it's a bit redundant, but that's not necessarily a bad thing here

Reply:
cleaned up!